### PR TITLE
adding html and css file

### PR DESCRIPTION
**Pull request by ibrahim777764**
please review my code

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+
+body{
+    display: flex;
+    justify-content: center;
```
Comment (ibrahim777764) [inline style.css:11]: This background image is not visible on browser, you can add it by adding correct path

File: index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+              name="age"
+              id="number"
+              min="10"
+              max="99"
+              class="form-control"
+              placeholder="Age"
+            />
+          </div>
+          <div>
+            <div>
```
Comment (ibrahim777764) [inline index.html:54]: please make sure your design aligns with the original design guidelines(fonts, colors, margins, paddings etc).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                <label for="checkbox">May be</label> 
+            </div>
+            <div class="text-area">
+                <label for="textarea" id="suggestions">Any Suggestions ? Write yours thoughts here :</label>
+                
+            </div>
+            <div id="button">
+                <textarea id="" cols="30" rows="7"></textarea>
+                <button type="button">Submit</button>
+            </div>
```
Comment (ibrahim777764) [inline index.html:89]: The button (e.g., "submit") is a link a or button tag, and follow the button styling: adjust width and height

**Review by ibrahim777764 — COMMENTED** [review]
Hi @ibrahim777764,
You have been doing a tremendous job so far ok_hand
There are some issues that you still need to work on to go to the next project but you are almost there!

### changed required :recycle: 

- [ ]   Add readme file , The readme file is used to explain what is uploaded and how we can install or use it.

Check the comments under the review.

Happy coding! :ok_hand:

**Comment by prolajumokeoni** [comment]
> Hi @ibrahim777764, You have been doing a tremendous job so far ok_hand There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ### changed required ♻️
> * [ ]    Add readme file , The readme file is used to explain what is uploaded and how we can install or use it.
> 
> Check the comments under the review.
> 
> Happy coding! 👌

- Good summary!
 You can also add your Github profile handle so the student can contact you if they don't understand a requested change 👍🏿

File: style.css (hunk 1 above)
Comment (prolajumokeoni) [reply style.css:11]: - [ ] Good job finding the broken link, Kindly make your comments more polite by using "Please" or "Kindly" in your comments 🤗

File: index.html (hunk 2 above)
Comment (prolajumokeoni) [reply index.html:54]: - [ ] Good job adding this polite comment, you can also state the importance of this change requested or why it was requested and kindly highlight a specific line or lines where changes are requested this help your comments be clear, easy to understand and give the student more direction 👍🏿

File: index.html (hunk 3 above)
Comment (prolajumokeoni) [reply index.html:89]: - [ ] It would be nice  to make your comments as suggestions instead of orders, doing this will make your comments polite 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- You added an inline comment ✔️ 
- You used the GitHub review changes feature ✔️  



## Required Changes ♻️
 
_Check the comments under the review._

 
154781434-6d53323e-228a-4d3b-87ae-852b2284b665

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ibrahim777764) [inline index.html:54]: - [ ] please make sure your design aligns with the original design guidelines(fonts, colors, margins, paddings etc).in this case your project cards are not almost exactly the original design 👍🏿

File: index.html (hunk 3 above)
Comment (ibrahim777764) [inline index.html:89]: - [ ] kindly,follow the button styling: adjust width and height :+1:

File: style.css [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    background-image: url(./workers.webp);
```
Comment (ibrahim777764) [inline style.css:12]: - [ ]  kindly change your image path ,This background image is not visible on browser, you can add it by adding correct path :

**Review by ibrahim777764 — COMMENTED** [review]
Hi @ibrahim777764,

Good job so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!

### Required Changes :recycle: 

- [ ]  Add readme file , The readme file is used to explain what is uploaded and how we can install or use it.

Cheers and Happy coding! :clap: :clap:  :clap: 

feel free to leave any questions or comments in the PR thread if something is not 100% clear.
@ibrahim777764 

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**Review by keroloussamy — APPROVED** [review]
Hi @ibrahim777764, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- Comments are polite.✔️
- Includes an explanation. ✔️
- Comments are clear and easy to understand. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**